frontend/src/routes/manage/Video/Single.tsx

```diff
@@ -0,0 +1,191 @@
+    const created = new Date(event.created).toLocaleString(i18n.language);
+    const updated = new Date(event.updated).toLocaleString(i18n.language);
+
+    const DateValue = useCallback(({ label, value }) => (
+        <div css={{ "&:not(:last-child)": { marginBottom: 12 } }}>
+            <div css={{ color: "var(--grey40)", lineHeight: 1 }}>{label + ":"}</div>
+            <div css={{ marginLeft: 6, marginTop: 4 }}>{value}</div>
+        </div>
+    ), []);
+
+    return (
+        <div css={{
+            flex: "0 0 auto",
+            display: "flex",
+            gap: 16,
+            [`@media(min-width: ${BREAKPOINT}px)`]: {
+                flexDirection: "column",
+                padding: 16,
+                borderLeft: "1px dashed var(--grey80)",
+            },
+        }}>
+            <Thumbnail event={event} width={16 * 12} />
+            <div css={{ fontSize: 14, margin: 4 }}>
+                {/* TODO: move those translation strings somewhere more appropriate */}
+                <DateValue label={t("video.created")} value={created} />
+                <DateValue label={t("video.updated")} value={updated} />
+            </div>
+        </div>
+    );
+};
+
+const MetadataSection: React.FC<Props> = ({ event }) => {
+    const { t } = useTranslation();
+
+    return (
+        <Form noValidate onSubmit={() => {}} css={{ margin: "0px 2px", flex: "1 0 auto" }}>
+            <InputContainer>
+                <TitleLabel htmlFor="title-field" />
+                <Input
+                    id="title-field"
+                    value={event.title}
+                    disabled
+                    css={{ width: "100%" }}
+                />
+            </InputContainer>
+
+            <InputContainer>
+                <label htmlFor="description-field">
+                    {t("upload.metadata.description")}
+                </label>
+                <TextArea id="description-field" disabled value={event.description ?? ""} />
+            </InputContainer>
+        </Form>
+    );
+};
+
+const TechnicalDetails: React.FC<Props> = ({ event }) => {
+    const { t } = useTranslation();
+
+    return <>
+        <h2 css={{ fontSize: 20, marginBottom: 8 }}>{t("manage.my-videos.technical-details")}</h2>
+        <div>
+            <span css={{ color: "var(--grey40)", marginRight: 8 }}>
+                {t("manage.my-videos.opencast-id") + ":"}
+            </span>
+            <code css={{ fontSize: 14 }}>{event.opencastId}</code>
+        </div>
+        <div css={{ marginTop: 8 }}>
+            <span css={{ color: "var(--grey40)", marginRight: 8 }}>
+                {t("manage.my-videos.available-resolutions") + ":"}
+            </span>
+            {event.tracks
+                .map(track => track.resolution)
+                .filter((r): r is number[] => r !== null)
+                .sort((a, b) => a[0] - b[0])
+                .map(r => r.join("x"))
+                .join(", ")
+            }
```

Comment:
```suggestion
                .join(", ")}
```

Reply:
That's not necessarily the style we use everywhere... e.g. look at `Login.tsx` at line 63 to 72. There are lots of these cases.

Reply:
Yikes, I hate it. :eyes:

Reply:
But ok, let's keep it for now, then. :shrug: